data/human/outfits.txt

```diff
@@ -391,6 +391,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Drugs"
```

Comment:
I  would prefer if the name is a bit more specific than just "Drugs".

Reply:
Would propose "Combat Anesthetic" or "Combat Stimulants" though those two classes of drugs tend to have opposite effects of eachother. Maybe avoid beating around the bush and call them "Combat Methamphetamines"?